_posts/2019-07-13-Going-Live.md

```diff
@@ -5,6 +5,10 @@ tags: Documentation
 categories: [Documentation]
 date: 2019-07-11 
 ---
-1. Change status of Issue
-2. Go to dashboard -> Settings -> Reading
-3. Update "Blog pages show at most" to match number of articles in the issue
\ No newline at end of file
+-> Go to Articles -> Issues -> IssueNumber -> Issue Status = Live
+
+-> Go to Articles -> Articles and then do the following
+	* Hover over the title of the article and set Status = Published and hit Update
```

Comment:
the bullet points are not aligned properly.

Reply:
I only copy-pasted the file given to me. I did not edit the file.
[Going_Live.txt](https://github.com/kmitemagazine/kmitemagazine.github.io/files/4380045/Going_Live.txt)